schema/rdl_schema_0.1.json

```diff
@@ -546,6 +545,37 @@
 		}
 	],
 	"$defs": {
+		"bibliography": {
+			"type": "object",
+			"description": "An array of one of more publications that relate to or have made use of this dataset",
```

Comment:
This definition describes a single publication rather than a list of publications so it should be called `publication` rather than `bibliography`.

Reply:
Continued in https://github.com/GFDRR/rdl-standard/issues/58